ntbs-service/Pages/Shared/_NotificationBanner.cshtml

```diff
@@ -19,61 +19,71 @@
         <nhs-grid-row classes="notification-overview-details">
             <nhs-grid-column grid-column-width="OneQuarter">
                 <h1 class="notification-banner-title-text">
-                    <div> @Model.Notification.NotificationStatusString </div>
-                    <div> #@Model.Notification.NotificationId </div>
+                        <div> @Model.NotificationStatusString </div>
+                    
+                @if(@Model.SearchPage) 

```

Comment:
`SearchPage` doesn't make sense from the abstraction of the banner class - we're letting the context leak into the implementation. Let's replace with something that describes the behaviour better without the need of context, e.g. `ShowLink`